pkg/virt-api/api.go

```diff
@@ -214,6 +214,7 @@ func (app *virtAPIApp) composeSubresources() {
 	for _, version := range v1.SubresourceGroupVersions {
 		subresourcesvmGVR := schema.GroupVersionResource{Group: version.Group, Version: version.Version, Resource: "virtualmachines"}
 		subresourcesvmiGVR := schema.GroupVersionResource{Group: version.Group, Version: version.Version, Resource: "virtualmachineinstances"}
+		expandvmspecGVR := schema.GroupVersionResource{Group: version.Group, Version: version.Version, Resource: "expand-vm-spec"}
```

Comment:
why was this named from `expand-spec` to `expand-vm-spec`?  what does adding "vm" improve?

Reply:
Since I'm already moving it to be namespaced, I considered Andrej's suggestion to rename it, so it hints that it takes a VM definition. Also keep in mind that this is a separate endpoint than the one below virtualmachines.